### PR TITLE
docs: worker v2 public docs.

### PR DESCRIPTION
**Pull request by davinchia**
## What
Update docs to prep for Worker V2 OSS launch.

The main focus is a brief explanation of what/why/how of the feature. We have a blog post in the works I'm going to link to for more explaination once it's published.

## How
* Remove old `*_WORKER` configs.
* Update the diagram to reflect the new flow. Viewing now is not great due to the rendering. Will likely follow up on a different diagram.
* Write Worker V2 explanation. Note that I left all the Docker pieces in place and tried to make things 'flow'. Will follow up here when we deprecate Docker officially.

## Review guide
- high-level-view.md: updated the flow diagram.
- jobs.md: most of the new changes.

## User Impact
No user impact.

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: docs/understanding-airbyte/high-level-view.md [hunk 1]
```diff
@@ -15,33 +15,43 @@ A more concrete diagram can be seen below:
...
-    W2 -->|launches| Source
-    W2 -->|launches| Destination
+    W2 -->|return status| T
+    W2 -->|creates job| WL
+    WL -->|queues workload| Q
+    Q  -->|reads from| L
+    L -->|launches| O
+    O -->|launches/reads from| Source
+    O -->|launches/reads from/writes to| Destination
+    O -->|reports status to| WL
```
Comment (davinchia) [inline docs/understanding-airbyte/high-level-view.md:41]: @Adorism any idea this looks pretty small on my browser.. any idea how to make this click & zoom?

Comment (davinchia) [reply docs/understanding-airbyte/high-level-view.md:41]: e.g. the rendered vercel: https://airbyte-docs-git-davinchia-worker-v2-public-airbyte-growth.vercel.app/understanding-airbyte/high-level-view

File: docs/understanding-airbyte/jobs.md [hunk 2]
```diff
@@ -312,33 +312,33 @@ The Cloud Storage store is treated as the source-of-truth of execution state.
...
-    Temporal->>Sync: Read results from Replication Activity
-    Sync->>Temporal: Start Persistent State Activity
-    Temporal->>PersistA: Pick up new task
-    PersistA->>AirbyteDB: Persist States
-    PersistA->>Temporal: Return output
-```
-
 Users running Airbyte Docker should be aware of the above pitfalls.
 
+## Worker V2
```
Comment (davinchia) [inline docs/understanding-airbyte/jobs.md:317]: @gosusnp here is where I would like you to focus on if you are short on time.

Once the blogpost is out, I'm going to link your post here at the bottom.

File: docs/understanding-airbyte/high-level-view.md (hunk 1 above)
Comment (davinchia) [reply docs/understanding-airbyte/high-level-view.md:42]: Using a different renderer solved this!

File: docs/understanding-airbyte/high-level-view.md [hunk 3]
```diff
@@ -15,33 +15,44 @@ A more concrete diagram can be seen below:
...
 - **Web App/UI** [`airbyte-webapp`, `airbyte-proxy`]: An easy-to-use graphical interface for interacting with the Airbyte API.
-- **Server/Config API** [`airbyte-server`, `airbyte-server-api`]: Handles connection between UI and API. Airbyte's main control plane. All operations in Airbyte such as creating sources, destinations, connections, managing configurations, etc.. are configured and invoked from the API.
+- **Config API Server** [`airbyte-server`, `airbyte-server-api`]: Handles connection between UI and API. Airbyte's main control plane. All operations in Airbyte such as creating sources, destinations, connections, managing configurations, etc.. are configured and invoked from the API.
 - **Database Config & Jobs** [`airbyte-db`]: Stores all the connections information \(credentials, frequency...\).
 - **Temporal Service** [`airbyte-temporal`]: Manages the task queue and workflows.
 - **Worker** [`airbyte-worker`]: The worker connects to a source connector, pulls the data and writes it to a destination.
+- **Workload API** [`airbyte-workload-api-server`]: Manages workloads, Airbyte's internal job abstraction.
+- **Launcher** [`airbyte-workload-launcher`]: Launch workloads.
```
Comment (gosusnp) [inline docs/understanding-airbyte/high-level-view.md:51]:
```suggestion
- **Launcher** [`airbyte-workload-launcher`]: Launches workloads.
```

Comment (gosusnp) [reply docs/understanding-airbyte/high-level-view.md:51]: We are a bit inconsistent on this, but at least we should try to align the paragraph.

File: docs/understanding-airbyte/jobs.md (hunk 2 above)
Comment (gosusnp) [reply docs/understanding-airbyte/jobs.md:317]: LGTM, the only comment is on the `Worker V2` name, should we just refer to this as `Workloads` instead?

**Review by gosusnp — APPROVED** [review]

**Pull request merged**